tripy/tripy/backend/compiler_api.py

```diff
@@ -31,6 +32,136 @@
 from tripy.common.shape_bounds import ShapeBounds
 from tripy.frontend import Tensor, Trace
 from tripy.utils import json as json_utils
+from tripy.backend.mlir.utils import MLIRRuntimeClient
+
+
+@export.public_api(document_under="compiler")
+class Stream:
+    """
+    Represents a CUDA stream that can be used to manage concurrent operations.
+
+    This class is a wrapper around the underlying stream object, allowing management of CUDA streams.
+
+    Args:
+        default_stream (bool): If True, uses the default stream. If False, creates a new stream.
```

Comment:
I meant that there should be a `tp.default_stream` or maybe `tp.Stream.default()` otherwise the semantics of this constructor are confusing.

Reply:
`tp.Stream.default_stream()` exists. Let me know if you `default` over `default_stream`?

Reply:
Let's remove the constructor argument then? No need to have two ways to access the default stream

Reply:
Fixed in latest iteration.